installation/windows.md

```diff
@@ -44,7 +53,7 @@ Assuming a successful install, you will now have various folders inside `C:\open
 ## Backup
```

Comment:
Additionally, Could you name the headline "File Locations" and bring everything in sync with http://docs.openhab.org/installation/linux.html#file-locations
That would be nice.